liquidity-amo/contracts/MasterAMO.sol

```diff
@@ -198,30 +190,26 @@ abstract contract MasterAMO is
      */
     function _mintSellFarm(
         uint256 boostAmount,
-        uint256 minUsdAmountOut,
         uint256 minBoostSpend,
-        uint256 minUsdSpend,
-        uint256 deadline
+        uint256 minUsdSpend
     )
         internal
         returns (uint256 boostAmountIn, uint256 usdAmountOut, uint256 boostSpent, uint256 usdSpent, uint256 liquidity)
     {
-        (boostAmountIn, usdAmountOut) = _mintAndSellBoost(boostAmount, minUsdAmountOut, deadline);
+        (boostAmountIn, usdAmountOut) = _mintAndSellBoost(boostAmount);
 
         uint256 price = boostPrice();
         if (price > FACTOR - validRangeWidth && price < FACTOR + validRangeWidth) {
             uint256 usdBalance = IERC20Upgradeable(usd).balanceOf(address(this));
-            (boostSpent, usdSpent, liquidity) = _addLiquidity(usdBalance, minBoostSpend, minUsdSpend, deadline);
+            (boostSpent, usdSpent, liquidity) = _addLiquidity(usdBalance, minBoostSpend, minUsdSpend);
         }
     }
 
     /// @inheritdoc IMasterAMO
     function mintSellFarm(
         uint256 boostAmount,
-        uint256 minUsdAmountOut,
```

Comment:
Just want to double check. The `minUsdAmountOut` slippage is removed here (and `minBoostAmountOut` removed in `_unfarmBuyBurn()` function). This is expected because the price check is within AMO specific implementations:
- V2AMO checks post-swap price so we don't overbuy or oversell
- V3AMO places a `targetSqrtPriceX96` limit price during swap

Reply:
Correct.